packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -491,3 +491,24 @@ new ec2.Instance(this, 'Instance', {
 });
 
 ```
+
+## User Data
+User data enables you to run a script when your instances start up.  In order to configure these scripts you can add commands directly to the script
+ or you can use the UserData's convenience functions to aid in the creation of your script.
+ 
+A user data could be configured to run a script found in an asset through the following:
+```ts
+const asset = new Asset(this, 'Asset', {path: path.join(__dirname, 'configure.sh')});
+const instance = new ec2.Instance(this, 'Instance', {
```

Comment:
Please fix the indentation and other whitespace here to at least be consistent.

Indentation of 2 spaces please, closing curly is not indented.